coderd/database/migrations/000155_workspace_agent_script.up.sql

```diff
@@ -0,0 +1,36 @@
+BEGIN;
+CREATE TABLE workspace_agent_log_sources (
+	workspace_agent_id uuid NOT NULL,
+	id uuid NOT NULL,
+	created_at timestamptz NOT NULL,
+	display_name varchar(127) NOT NULL,
+	icon text NOT NULL,
+	PRIMARY KEY (workspace_agent_id, id)
+);
+
+CREATE TABLE workspace_agent_scripts (
+	workspace_agent_id uuid NOT NULL,
+	log_source_id uuid NOT NULL,
+	log_path text NOT NULL,
+	created_at timestamptz NOT NULL,
+	source text NOT NULL,
+	cron text NOT NULL,
+	start_blocks_login boolean NOT NULL,
+	run_on_start boolean NOT NULL,
+	run_on_stop boolean NOT NULL,
+	timeout integer NOT NULL
```

Comment:
This'll help with discerning the unit.

```suggestion
	timeout_seconds integer NOT NULL
```